lib/archethic/bootstrap.ex

```diff
@@ -206,12 +208,32 @@ defmodule Archethic.Bootstrap do
       Logger.info("Synchronization finished")
     end
 
+    resync_network_chain([:oracle, :node_shared_secrets])
+
     Sync.publish_end_of_sync()
     SelfRepair.start_scheduler()
-
     :persistent_term.put(:archethic_up, :up)
   end
 
+  def resync_network_chain(type_list) do
+    Task.Supervisor.async_stream_nolink(Archethic.TaskSupervisor, type_list, fn type ->
+      with nodes <- P2P.authorized_and_available_nodes(),
```

Comment:
Maybe something like to avoid `error` == nil

```elixir
if TransactionChain.list_addresses_by(type) |> Enum.at(0) do
  with {:ok, last_addr} <- TransactionChain.fetch_last_address_remotely(addr, nodes),
       {:ok, last_tx} <- TransactionChain.fetch_transaction_remotely(last_addr, nodes),
       :ok <- Replication.validate_and_store_transaction_chain(last_tx) do
      Logger.info("Resync Success", transaction_type: type)
   end
end
```